packages/reference/src/components/LinkActions/CombinedLinkActions.tsx

```diff
@@ -38,7 +38,7 @@ function CombinedEntryLinkActions(props: LinkActionsProps) {
       <CreateEntryLinkButton
         testId={testIds.actionsWrapper}
         disabled={props.isDisabled}
-        text="Add content"
+        text={props.combinedActionsLabel}
```

Comment:
should the text have a default value of "Add content" if combinedActionsLabel is undefined?